### DISCUSSION TITLE
ARM-based CPU

### DISCUSSION THREAD

**Discussion by weisselbiad** [manoelcampos/cloudsim-plus#377] [General]
Hey,
Is there any way to model an ARM-based CPU architecture ? and how can i convert MIPS to Flops ?
Thanks

**Comment by jlxcsong** [comment]
您好！您发的邮件我已收到！

**Comment by manoelcampos** [comment]
Hello @weisselbiad,
CloudSim Plus doesn't model processors at such a low level. It doesn't implement any specific CPU architecture.
The MIPS unit is used as a way to define the CPU speed. This way, the length of processes (cloudlets) is defined in MI.
But if you want to ignore that the units are MIPS and MI and use **Float-Point Operations/Second** for CPUs and **Float-Point Operations** for Cloudlet length instead, there is no problem.
